src/scrape_up/devCommunity/articles.py

```diff
@@ -0,0 +1,433 @@
+from bs4 import BeautifulSoup
+from pprint import pprint
+import json
+import requests
+
+class DevCommunity:
+    """
+    Class - `DevCommunity` \n
+
+    Creates a scraper for latest articles from https://www.dev.to.
+    Creates a scraper for latest articles from https://www.dev.to/t/{tag} with a given tag.
+    Creates a scraper for  user details, pinned articles written by a user, and all articles 
+    written by a user from https://www.dev.to/{username}.
+
+    Example - \n
+    ```python
+    dev = DevCommunity('python','francescoxx')
+    """
+    def __init__(self, tag: str, username: str):
+        self.help = "This scrapes articles from the DevCommunity website."
+        self.tag = tag
+        self.username = username
```

Comment:
Including `tags=` as a constructor is not a good idea I guess, as it might not be required in a few methods.
Instead, include it in those methods only which is required.